app/models/financialDetails/BalanceDetails.scala

```diff
@@ -35,6 +36,10 @@ case class BalanceDetails(balanceDueWithin30Days: BigDecimal,
     total - firstPendingAmountRequested.getOrElse(0) - secondPendingAmountRequested.getOrElse(0)
   }
 
+  def getAvailableCredit: BigDecimal = {
```

Comment:
I don't think we need this change, we already have these defined in WhatYouOweChargesList or we should define in other viewModels as a part of future refactoring;